bin/cfgyaml2make

```diff
@@ -48,6 +48,7 @@ TOPDIR = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)
 REQUIRED_KEYS = ('name', 'description',)
 CFG_PATHS = (
              '<CV_CORE>/tests/cfg',
+             '<CV_CORE>/tests/test_cfg',
```

Comment:
After a second reading, I am wondering if this can cause issues, because the main CFG= option will also look inside tests/test_cfg

Reply:
The cfg/test_cfg file names are expected to be unique, and incase multiple cfg files are found with same names, the script already flags a fatal error. So any one creating a new config and running a check for those new configs should be able to hit this issue and create a unique name for new cfg file.
And ensuring the regress list is created with correct cfg/test_cfg is always the responsibilty of the dv which ensures we are creating correct scenario. 
@XavierAubert  do you still see need for 2 separate yaml2make scripts for these 2 ?

Reply:
yes, true I did not see it was also checking for duplicates.